e2e/tools/validator/query.json

```diff
@@ -0,0 +1,43 @@
+{
```

Comment:
Double check this

Reply:
ISTR this file was removed recently and replaced with a YAML config file